assignment-2/src/model/modelQuestion.js

```diff
@@ -0,0 +1,128 @@
+import { EventEmitter } from "events";
+import RestClient from "../rest/RestClient";
+
+const client= new RestClient("ioana","poop");
```

Comment:
Interesting password. Also, you are always hard-coding your username / password.